src/js/stores/NotificationStore.jsx

```diff
@@ -66,6 +77,15 @@ class NotificationStore extends EventEmitter {
     self.notifications = self.notifications.filter((notification) => {
       return notification.id != notificationParam.id;
     });
+
+    // Set this notification as viewed so it doesn't appear again
+    let viewedNotifications = JSON.parse(localStorage.getItem("viewedNotifications"));
```

Comment:
@schneider-clarisse I missed removing the parse from the constructor which is my mistake. We do parse twice in the code. The reasoning I had behind this was that every time we read or want to append to it we should make sure they have the newest version of this list. (They may have it open in multiple chrome windows and keep updating it from these different windows. If that's the case we want to be able to reflect these changes across all windows). 

Although I guess this still wouldn't cover all instances :confused: . I can move it back to the constructor if you think that's better.